Python/optimizer.c

```diff
@@ -195,7 +182,7 @@ _Py_SetOptimizer(PyInterpreterState *interp, _PyOptimizerObject *optimizer)
 }
 
 int
-PyUnstable_SetOptimizer(_PyOptimizerObject *optimizer)
+_Py_SetOptimizerAPI(_PyOptimizerObject *optimizer)
```

Comment:
Why the `API` suffix?
Maybe `Py_SetTier2Optimizer` (to distinguish it from the AST, bytecode or tier 1 optimizers)

Reply:
There is another _Py_SetOptimizer() function with a different API. This one adds more boilerplate code and calls the first one.

Reply:
I renamed the function to `_Py_SetTier2Optimizer()` (I added an underscore prefix).